InfluxPublisher/README.md

```diff
@@ -6,15 +6,15 @@ This folder contains Lingua Franca reactors that publish to InfluxDB via a socke
 
 1. Compile the LF example applications:
 
-    ```bash
-    cd c && lfc src/InfluxPublisher.lf
-    cd py && lfc src/InfluxPublisher.lf
-    ```
+   ```bash
+   cd c && lfc src/InfluxPublisher.lf; cd ..
```

Comment:
Ah, good cathch. I prefer `(cd c && lfc src/InfluxPublisher.lf)` since it's easier to combine with other commands, but just food for thought.

Reply:
Sounds good, README.md has been updated with the improved Bash commands.